client/js/pages/archive/index.js

```diff
@@ -1,14 +1,10 @@
 const html = require('choo/html')
```

Comment:
Yes, but i think we should diverge the render-ness logic in the component level rather than the entry-point to make things simpler.  for example: the hyperdrive-ui bit can be rendering the html tree plus a tree object in server, while in client side it's rehydrated with yofs on the tree object, and connects to peer for updates.  Does it make sense?